core/App/components/misc/CredentialCard.tsx

```diff
@@ -69,29 +69,42 @@ const CredentialCard: React.FC<CredentialCardProps> = ({ credential, style = {},
 
   const styles = StyleSheet.create({
     container: {
-      backgroundColor: bundle
-        ? overlay?.imageSource
-          ? transparent
-          : overlay?.backgroundColor ?? credentialBackgroundColor
-        : transparent,
+      backgroundColor: overlay?.imageSource ? transparent : overlay?.backgroundColor,
       height: width / 2,
       borderRadius: borderRadius,
+      //borderColor: 'black',
+      //borderStyle: 'solid',
+      //borderWidth: 1,
     },
-    headerContainer: {
-      flexDirection: 'row',
-      backgroundColor: bundle ? overlay?.header?.backgroundColor || transparent : transparent,
-      height: width / 7.5,
-      paddingHorizontal,
-      paddingVertical,
+    outerHeaderContainer: {
+      flexDirection: 'column',
+      backgroundColor: overlay?.header?.backgroundColor || transparent,
+      // eslint-disable-next-line prettier/prettier
+      height: (width / 2 / 4) +  (borderPadding),
+      //paddingHorizontal,
+      //paddingVertical,
       borderTopLeftRadius: borderRadius,
       borderTopRightRadius: borderRadius,
+      //paddingLeft: 10,
+      //paddingRight: 10,
+      //paddingTop: 10,
+    },
+    innerHeaderContainer: {
+      flexDirection: 'row',
+      // eslint-disable-next-line prettier/prettier
+      height: (width / 2 / 4),
```

Comment:
Why do we need the double division here. If those are constants, can we name them accordingly and perhaps maybe use parentheses for the denominator?

Reply:
Added some docs (as comments)